frontend/src/components/Friend/Friend.jsx

```diff
@@ -0,0 +1,45 @@
+import "./Friend.css";
+import axios from "axios";
+import { useContext } from "react";
+import { UserContext } from "../App/App";
+
+export default function Friend({ username, profile, setProfile }) {
```

Comment:
I'd recommend calling this something like `FriendRequest` or similar, since this component is mainly for the processing of "friending" since it is a wrapper around the button

Reply:
FirendConnect or FriendConnection or similar. We need to know the distinction between this and a request-aprove|reject mechanism